src/patternfly/components/Popover/popover-arrow.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-popover__arrow{{#if popover-body--modifier}} {{popover-body--modifier}}{{/if}}"
```

Comment:
these should be arrow not body for the modifier and attribute.

Reply:
doh! fixed.